OJS.Workers.SubmissionProcessors/Helpers/SubmissionProcessorHelper.cs

```diff
@@ -381,35 +341,6 @@ public static IExecutionStrategy CreateExecutionStrategy(ExecutionStrategyType t
                         Settings.PhpCliBaseTimeUsedInMilliseconds,
                         Settings.PhpCliBaseMemoryUsedInBytes);
                     break;
-                case ExecutionStrategyType.SolidityCompileDeployAndRunUnitTestsExecutionStrategy:
-                    executionStrategy = new SolidityCompileDeployAndRunUnitTestsExecutionStrategy(
-                        GetCompilerPath,
-                        processExecutorFactory,
-                        Settings.NodeJsExecutablePath,
-                        Settings.GanacheCliNodeExecutablePath,
-                        Settings.TruffleCliNodeExecutablePath,
-                        int.Parse(submissionProcessorIdentifier),
-                        Settings.SolidityBaseTimeUsedInMilliseconds,
-                        Settings.SolidityBaseMemoryUsedInBytes);
```

Comment:
These settings should be removed if not used. For the other strategies as well.

Reply:
For the solidity they are removed , but for the other strategies it seems that the settings are in use

Reply:
Where are they used? Investigate the usage and see if they are really used. For example SolidityBaseMemoryUsedInBytes is not removed from Settings

Reply:
DotNet time and memory settings are used in other DotNet strategies,
NUnit is used in CSharpUnitTests strategy,
MsBuild time and memory settings are used in other CSharp strategies,
PhpCli time and memory settings are used in PhpCli and PhpCgi strategies

 Settings.MySqlSysDbConnectionString, Settings.MySqlRestrictedUserId,Settings.MySqlRestrictedUserPassword, these are used in the MySQL strategies.

I thought that i removed the memory and time settings for the solidity, but apparently i forgot, so  will remove them now

Reply:
There is `GanacheCliDefaultPortNumber`. Can you remove it as well